compiler/damlc/tests/daml-test-files/ExportList.EXPECTED.rst

```diff
@@ -3,6 +3,105 @@
 Module ExportList
 -----------------
 
+Templates
+^^^^^^^^^
+
+.. _type-exportlist-template0-22237:
+
+**template** `Template0 <type-exportlist-template0-22237_>`_
+
+  .. list-table::
+     :widths: 15 10 30
+     :header-rows: 1
+  
+     * - Field
+       - Type
+       - Description
+     * - tfield0
+       - Party
+       - 
+     * - tfield0'
+       - Party
+       - 
+  
+  + **Choice External\:Archive**
+    
+  
+  + **Choice Choice0**
+    
+
+.. _type-exportlist-template1-34256:
+
+**template** `Template1 <type-exportlist-template1-34256_>`_
+
+  .. list-table::
+     :widths: 15 10 30
+     :header-rows: 1
+  
+     * - Field
+       - Type
+       - Description
+     * - tfield1
+       - Party
+       - 
+     * - tfield1'
+       - Party
+       - 
+  
+  + **Choice External\:Archive**
+    
+  
+  + **Choice Choice1**
+    
+
+.. _type-exportlist-template2-3915:
+
+**template** `Template2 <type-exportlist-template2-3915_>`_
+
+  .. list-table::
+     :widths: 15 10 30
+     :header-rows: 1
+  
+     * - Field
+       - Type
+       - Description
+     * - tfield2
+       - Party
+       - 
+     * - tfield2'
+       - Party
+       - 
+  
+  + **Choice External\:Archive**
```

Comment:
Do we use `External` for anything coming from an external package? I think we might want to special case this for `Archive` given how common it is.

Reply:
Good point, worth a special case.
When I added this, the desugaring for templates was different, and was used in our code for what is now achieved by generic template instances.

Reply:
I'll remove it in a separate PR.